net/sixlowpan/sixlowpan_udpsend.c

```diff
@@ -389,11 +389,11 @@ ssize_t psock_6lowpan_udp_send(FAR struct socket *psock, FAR const void *buf,
 
   /* Ignore if not IPv6 domain */
 
-  if (conn->domain != PF_INET6)
```

Comment:
is this commenting out necessary? It seems like it should only check for IPv6, which would be good to do, right?

Reply:
Well about this point, we can consider a temporary fix. Because I've looked for information about the difference between AF_INET and PF_INET and it supposes that are equivalent.
So, the send function call to the [psock_6lowpan_udp_sendto](https://github.com/micro-ROS/NuttX/blob/1a82d5907cd6d1686f327ec582053bb502833bb1/net/sixlowpan/sixlowpan_udpsend.c#L404) which already check if we're using an AF_INET6 address family [L195](https://github.com/micro-ROS/NuttX/blob/1a82d5907cd6d1686f327ec582053bb502833bb1/net/sixlowpan/sixlowpan_udpsend.c#L195).